### PR TITLE
[6.16.z]: Changes for 6.16.z new branch

### PR DESCRIPTION
**Pull request by Satellite-QE**
### Problem Statement
  New 6.16.z branch
  ### Solution
  - Dependabot labels are updated for new branch
  - Removed dispatch release GHA from 6.16.z as we are releasing only master changes
  - Airgun and Nailgun Requirements uses 6.16.z branch
  - Constants are using new version now
  - Stream tests removed
  - Setup.py uses new version

### CODE REVIEW

**Comment by Satellite-QE** [comment]
trigger: test-robottelo

**Review by jyejare — APPROVED** [review]

**Comment by omkarkhatavkar** [comment]
CI checks are failing because we need the Airgun and Nailgun branching as well

**Review by vsedmik — APPROVED** [review]
ACK on the `@pytest.mark.stream` removal.

**Comment by jyejare** [comment]
@omkarkhatavkar Here are the:

Nailgun PRs:
-------------
https://github.com/SatelliteQE/airgun/pull/1407 
https://github.com/SatelliteQE/airgun/pull/1406 


Airgun PRs:
------------
https://github.com/SatelliteQE/nailgun/pull/1190
https://github.com/SatelliteQE/nailgun/pull/1189

File: .github/dependabot.yml [hunk 1]
```diff
@@ -23,6 +24,7 @@ updates:
     schedule:
       interval: "daily"
     labels:
+      - '6.16.z'
```
Comment (Gauravtalreja1) [inline .github/dependabot.yml:27]: Can we remove 6.12.z from here as well?

Comment (jyejare) [reply .github/dependabot.yml:27]: @Gauravtalreja1 Lets not hold this PR for 6.12 changes, we can tackle that in a separate PR.

Comment (omkarkhatavkar) [reply .github/dependabot.yml:26]: @jyejare @Gauravtalreja1 updated

Comment (Gauravtalreja1) [reply .github/dependabot.yml:26]: @omkarkhatavkar Thank you!

**Pull request closed without merging**